src/selectors/Entities/name.js

```diff
@@ -87,3 +87,11 @@ export const selectVaccinePatientHistory = state => {
   const fullQuery = `(${inQuery}) AND ${baseQueryString} AND itemBatch.item.isVaccine == true`;
   return inQuery ? UIDatabase.objects('TransactionBatch').filtered(fullQuery).slice() : [];
 };
+
+export const selectWasPatientVaccinatedToday = state => {
+  const history = selectVaccinePatientHistory(state);
+
+  const currentDate = new Date();
+  return !!history.filter(historyRecord => historyRecord.confirmDate >= currentDate.getDate() - 1)
+    .length;
```

Comment:
```suggestion
  const history = selectVaccinePatientHistory(state);
  const currentDate = new Date().getTime() - MILLISECONDS_PER_DAY;

  return !!history.filter(historyRecord => historyRecord.confirmDate.getTime() > currentDate)
    .length;
```

Reply:
Thanks, I took your suggestion and got a bit carried away with the modalString and variable names 😅  2217ced